string[] state
string[] actions
string reward
---
bool configured
int64 state_dim
int64 actions_dim
